src/test/java/com/uipath/uipathpackage/UiPathRunJobTests.java

```diff
@@ -0,0 +1,238 @@
+                orchestratorAddress, orchestratorTenant, folderName, userPassCredentials);
+
+        project.getPublishersList().add(publisher);
+        project = jenkins.configRoundtrip(project);
+
+        jenkins.assertEqualDataBoundBeans(new UiPathRunJob(defaultProcessName, "", StartProcessDtoJobPriority.Normal, defaultDynamicallyStrategy, "test", null, true,
+                orchestratorAddress, orchestratorTenant, folderName, userPassCredentials), project.getPublishersList().get(0));
+    }
+
+    @Test
+    public void runJobWithTimeout() throws Exception {
+        UiPathRunJob publisher = new UiPathRunJob(defaultProcessName, "", StartProcessDtoJobPriority.Normal, defaultDynamicallyStrategy, "", 10000, true,
+                orchestratorAddress, orchestratorTenant, folderName, userPassCredentials);
+
+        project.getPublishersList().add(publisher);
+        project = jenkins.configRoundtrip(project);
+
+        jenkins.assertEqualDataBoundBeans(new UiPathRunJob(defaultProcessName, "", StartProcessDtoJobPriority.Normal, defaultDynamicallyStrategy, "", 10000, true,
+                orchestratorAddress, orchestratorTenant, folderName, userPassCredentials), project.getPublishersList().get(0));
+    }
+
+    @Test
+    public void runJobClassicFolder() throws Exception {
+        UiPathRunJob publisher = new UiPathRunJob(classicProcessName, null, StartProcessDtoJobPriority.High, defaultDynamicallyStrategy, null, 10000, true,
+                orchestratorAddress, orchestratorTenant, folderName, userPassCredentials);
+
+        project.getPublishersList().add(publisher);
+        FreeStyleBuild build = jenkins.buildAndAssertSuccess(project);
+
+        jenkins.assertLogContains("Starting job run", build);
+        jenkins.assertLogContains("Finished running job for process with id", build);
+        jenkins.assertLogContains("Running jobs... 1 passed, 0 stopped or terminated, 0 total. Waiting for 20s.", build);
+    }
+
+    @Test
+    public void runJobClassicFolderWithSpecificRobots() throws Exception {
+        UiPathRunJob publisher = new UiPathRunJob(classicProcessName, null, StartProcessDtoJobPriority.High, robotStrategy, null, 10000, true,
+                orchestratorAddress, orchestratorTenant, folderName, userPassCredentials);
+
+        project.getPublishersList().add(publisher);
+        FreeStyleBuild build = jenkins.buildAndAssertSuccess(project);
+
+        jenkins.assertLogContains("Starting job run", build);
+        jenkins.assertLogContains("Finished running job for process with id", build);
+        jenkins.assertLogContains("Running jobs... 1 passed, 0 stopped or terminated, 0 total. Waiting for 20s.", build);
+    }
+
+    @Test
+    public void runJobModernFolder() throws Exception {
+        UiPathRunJob publisher = new UiPathRunJob(classicProcessName, null, StartProcessDtoJobPriority.High, defaultDynamicallyStrategy, null, 10000, true,
```

Comment:
This is meant to run in a modern folder, so I guess you used the wrong process name.